usaspending_api/recipient/management/sql/recipient_lookup/030_sam_parents.sql

```diff
@@ -10,7 +10,7 @@ INSERT INTO public.temporary_restock_recipient_lookup (
 )
 SELECT
   DISTINCT ON (ultimate_parent_unique_ide, ultimate_parent_legal_enti)
-  MD5(UPPER(CONCAT('duns-', ultimate_parent_unique_ide)))::uuid AS recipient_hash,
+  MD5(COALESCE(UPPER(CONCAT('uei-', ultimate_parent_uei)), UPPER(CONCAT('duns-', ultimate_parent_unique_ide))))::uuid AS recipient_hash,
```

Comment:
I think the `010` step, which build the hash on the DUNS number, and `070` step, which builds the hash on the parent DUNS number are also missing from this PR. 
- To follow this pattern, wouldn't those build it on `uei` if available (for 010), and `ultimate_parent_uei` if available (for `070`)

In general though, i'm not sure if this is the pattern to follow. We may just need to add more steps for uei fields in this layered build-up of recipient records from available hash-worthy fields in an order-of-precedence. -- More in the review comment.